CHANGELOG.md

```diff
@@ -55,6 +55,9 @@ All notable changes to this project will be documented in this file.
   * #### Changed
     * Added default values for timeout on all fetch and read functions.
   * #### Removed
+    * `AddWaveformProcessing`
```

Comment:
Please add "See #667" for clients looking for the rationale behind this.

Reply:
Added